add animation queue publishing (#69)
husarion/panther_ros: panther_lights/README.md

- `/panther/lights/driver/front_panel_frame` [*sensor_msgs/Image*, encoding: **RGBA8**, height: **1**, width: **num_led**]: an animation frame pixels to be displayed on robot front LED panel.
- `/panther/lights/driver/rear_panel_frame` [*sensor_msgs/Image*, encoding: **RGBA8**, height: **1**, width: **num_led**]: an animation frame pixels to be displayed on robot rear LED panel.

> - `/panther/lights/controller/queue` [*panther_msgs/LEDAnimationQueue*]: list with names of animations that are being processed by controller node, the first element of the list is currently displayed animation.

Reviewer: ```suggestion
- `/panther/lights/controller/queue` [*panther_msgs/LEDAnimationQueue*]: list of names of currently enqueued animations in controller node, the first element of the list is the currently displayed animation.
```
